src/App.js

```diff
@@ -1,10 +1,43 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
+import Button from './Components/Button';
 import './App.css';
 
 function App() {
+  const [color, setColor] = useState({
+    addButton: "rgb(26, 94, 102)", 
+    viewButton: "rgb(26, 94, 102)"
+  });
+
+  const handleClick = (button) => {
+    if(button === "add") {
+      setColor({
+        addButton: "rgb(10, 177, 173)",
+        viewButton: "rgb(26, 94, 102)"
+      });
+    }
+    else if (button === "view") {
+      setColor({
+        addButton: "rgb(26, 94, 102)",
+        viewButton: "rgb(10, 177, 173)"
+      });
+    }
```

Comment:
There is no need to implement color-changing logic here. It should be handled by the component itself. We can just pass a props to handle it inside the component.

eg:
In parent component:
```
 <Button 
        label="Add Task" 
        onClick={() => handleClick("add")}
        backgroundColor={color.addButton} 
        isActive={true}
      />
```

In Button component:
const Button = ({label, backgroundColor, onClick,isActive = false, isDisabled = false}) => {
return (
       <button onClick={onClick} style={{ backgroundColor: isActive 'color1' : 'color2'}} disabled={isDisabled}> 
        {label}
        </button>
    )
}